### ISSUE TITLE
OrangeBits LESS compiling adding extra spaces where it shouldn't be.

### ISSUE THREAD

**Issue by itsnydell** [JustinBeckwith/OrangeBits#5]
In my .less file I have my font-family line:

``` css
font-family: MuseoSans-500, "Helvetica Neue", "HelveticaNeue", Helvetica, Arial, "Lucida Grande", sans-serif;
```

In my .css file after compilation it shows:

``` css
font-family: MuseoSans- 500, "Helvetica Neue", "HelveticaNeue", Helvetica, Arial, "Lucida Grande", sans-serif;
```

Notice the space between "-" and the "500". This causes the font to not display because it's not a correct font.

**Comment by JustinBeckwith** [comment]
Yikes.  For my underlying LESS compiler I use dotless --> https://github.com/dotless/dotless  Let me see if I can reproduce this independent of the extension - my guess is there's a bug with their engine.  I'll follow up soon.

**Comment by JustinBeckwith** [comment]
I reported the issue over at dotless:

https://github.com/dotless/dotless/issues/224

As a work around (until there's a fix), wrapping the font name in quotes seems to fix the problem.

**Comment by JustinBeckwith** [comment]
I just released OrangeBits 1.1.10, which should fix this problem.  I moved away from dotless to just running node to do the less compilation.  Please let me know if you still run into the problem with 1.0.10.
